appinfo/info.xml

```diff
@@ -22,7 +22,7 @@ Negative:
 
 Learn more about the Nextcloud Ethical AI Rating [in our blog](https://nextcloud.com/blog/nextcloud-ethical-ai-rating/).
     ]]></description>
-    <version>1.1.0</version>
+    <version>1.1.1</version>
```

Comment:
Dropping support for NC versions should bump the major version: https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/release_process.html#major-update